pkg/db/fetch/fetch.go

```diff
@@ -206,7 +206,6 @@ func (o *options) finish(dbpath, digest string) error {
 	if err := dbc.Open(); err != nil {
 		return errors.Wrap(err, "db open")
 	}
-	defer dbc.Close() //nolint:errcheck
```

Comment:
By removing this line, dbc may be left opened in some error cases, e.g. "get metadata" at L.212 ?

Reply:
If Close() can be called multiple times (it should be, I think), we can retain defer after adding Close() at L228?